src/godot_fmod.h

```diff
@@ -57,7 +57,7 @@ namespace godot {
 #define FIND_AND_CHECK_WITH_RETURN(instanceId, cont, defaultReturn) \
     auto instance = (cont).get(instanceId); \
     if (!instance) { \
-        String message = String("FMOD Sound System: cannot find " + String(instanceId) + " in ##cont collection.");\
```

Comment:
We sometimes inject non-String types into the Macro and in Godot 4 this is not possible any longer to "just cast" the instanceId type. For now I removed the ID from the logging but if someone has an idea how to bring it back "generically" that would be great.